epub33/a11y/index.html

```diff
@@ -915,83 +925,96 @@ <h6>Skippability</h6>
 										colours or placement so readers can filter this information visually out while
 										reading).</p>
 									<p>Readers who prefer auditory playback, however, cannot skip this information with
-										the same ease by default. Media overlays playback will typically result in
-										Reading Systems rendering secondary content where it occurs.</p>
-									<p>When EPUB Creators add structural semantics to Media Overlay Documents, however,
-										Reading Systems can create reading experiences that allow users to decide which
-										secondary content to skip by default during playback.</p>
+										the same ease by default. Synchronized text-audio playback will typically result
```

Comment:
Suggested change:

"Readers who prefer auditory playback, however, cannot skip this information with the same ease in a linear audio-based reading experience. And, without structural semantics, synchronized text-audio playback cannot offer skipping content either."